packages/addon-dev/sample-typescript-babel.config.cjs

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { resolve } = require;
+
+module.exports = {
+  presets: [resolve('@babel/preset-typescript')],
```

Comment:
Using resolve here, because I hate "plugin not found errors" in babel. Typos are much easier to catch earlier, and resolve helps out with that.

Reply:
Why do we have both `@babel/preset-typescript` and `@babel/plugin-transform-typescript`? I think it should be one or the other. The only thing the preset does is install that plugin.

Reply:
plugin-transform-typescript is required when doing `resolve` in the babel.config.js (by eslint + node:recommended)
if I just use strings for the plugin names, the issue is side-stepped.